CRM/UF/Page/ProfileEditor.php

```diff
@@ -20,18 +20,26 @@ public static function registerProfileScripts() {
 
     CRM_Core_Resources::singleton()
       ->addSettingsFactory(function () {
+        $ufGroups = civicrm_api3('UFGroup', 'get', array(
+          'sequential' => 1,
+          'is_active' => 1,
+          'rowCount' => 1000, // FIXME
```

Comment:
yes, it works, created a new PR at https://github.com/civicrm/civicrm-core/pull/6336

Thanks for the suggestion.